authz/authz.proto

```diff
@@ -247,10 +255,20 @@ message ProbeResponse {
 
   Action action = 1;
   string version = 2;
+  string authz_profile_id = 3;
 }
 
 // GetRequest used to request the gRPC-level Authorization Policy.
 message GetRequest {
+  // The profile for which the authz policy is being requested. In the case that
+  // this field is not specified, the default authz policy which applies to all
+  // gRPC endpoints run by the target is assumed. Where non-default policies
+  // are to supported by an endpoint, the value of the profile determines which
+  // set of policies are returned.
+  //
+  // Note that the authz profile is considered independent from a SSL profile
+  // ID (as referenced by gnsi.Certz).
+  string authz_profile_id = 1;
```

Comment:
Is this really 'ssl_profile_id'? or are there now new profiles inside profiles?
(authz inside ssl)

wait: "Note that the authz profile is considered..."

Why is that? are these tied back to the ssl profile in some way? how would I know that I'm using the a vs b profile here?